### DISCUSSION TITLE
Seurat scRNA integrated vs RNA assay

### DISCUSSION THREAD

**Discussion by LacquerHed** [GreenleafLab/ArchR#1661] [Questions / Documentation] [answered]
Since I used the integrated assay for clustering in Seurat are the variable genes from this the appropriate feed-in to the ArchR integration or should it be from the Seurat RNA assay? 

Also not quite sure what unmodified Seurat objects are, do SCT normalized objects prior to integration and clustering in Seurat count as "modified"?

**Comment by rcorces** [comment, marked as answer]
I'm not following your first question.

> Also not quite sure what unmodified Seurat objects are, do SCT normalized objects prior to integration and clustering in Seurat count as "modified"?

As per Seurat guidelines, SCT normalized data is not meant to be used for integration

**Reply by LacquerHed** [reply]
For the first question just curious if I should use integrated RNA datasets (non-SCT transformed or any other mods) or just simply merged ones as the feed in to ArchR integration?

**Reply by rcorces** [reply]
Sorry if I'm being dense but this still doesnt make sense to me. Perhaps you can look at the RNA object provided in the tutorial.

**Reply by LacquerHed** [reply]
I figured out that I was looking at it incorrectly, thanks for the help!

**Comment by dimurali93** [comment]
I also have the same question. could you to point me where to look @LacquerHed ?
